VeganVirus/VeganVirus/SoundAction.cpp

```diff
@@ -4,6 +4,7 @@ SoundAction::SoundAction(double req, AudioManager& audioManager)
     : Action(req, MUSIC_ICON_PATH), _audioManager(audioManager)
 {
     _timer = SOUND_DUARTION + 1;
+    start();
```

Comment:
You shouldn't call start in the constructor.
Did you start music at the start of the run to debug? 😏